src/app/modules/item/components/item-dependencies/item-dependencies.component.ts

```diff
@@ -74,4 +86,27 @@ export class ItemDependenciesComponent implements OnChanges, OnDestroy {
   refresh(): void {
     this.refresh$.next();
   }
+
+  addDependency(item: AddedContent<ItemType>): void {
+    if (!item.id) {
+      throw new Error('Unexpected: Missed item ID');
+    }
+    const dependentItemId = this.itemData?.item.id;
+    if (!dependentItemId) {
+      throw new Error('Unexpected: Missed dependent item id');
```

Comment:
For the error messages, I'd say: `throw new Error('Unexpected: item id is missing')` and `throw new Error('Unexpected: dependent item id is missing')`

Reply:
Fixed